packages/adp-tooling/templates/rta/controller.ejs

```diff
@@ -0,0 +1,86 @@
+/*!
```

Comment:
That one doesn't belong into this PR, correct? At least I didn't see any code generating a controller

Reply:
Removed in: https://github.com/SAP/open-ux-tools/pull/1183/commits/4a2829c286592f1d845bdb93f987f25507c02025